src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -480,36 +482,31 @@ export default class rd2EntryForm extends LightningElement {
      * Keep Save button enabled so user can correct a value and save again.
      */
     handleSaveError(error) {
-        this.handleError(error, false);
+        this.error = constructErrorMessage(error);
+        this.setSaveButtonDisabled(false);
     }
 
     /***
      * @description Handle error and disable the Save button
      */
     handleError(error) {
-        this.handleError(error, true);
-    }
-
-    /***
-    * @description Handle component display when an error occurs
-    * @param error: Error Event
-    * @param disableSaveButton: Indicates if the Save button should be disabled
-    */
-    handleError(error, disableSaveButton) {
-        this.error = constructErrorMessage(error);
+        this.error = (error && error.detail)
+            ? error
+            : constructErrorMessage(error);
 
-        this.handlePageOnError(disableSaveButton);
+        this.setSaveButtonDisabled(true);
     }
 
     /***
     * @description Handle component display when an error occurs
-    * @param disableSaveButton: Indicates if the Save button should be disabled
+    * @param isDisabled: Indicates if the Save button should be disabled
```

Comment:
This is a real nitpick, but isDisabled sounds like the button is already disabled, however that is the result of this action.  I'm not sure that isDisabled is an improvement on disableSaveButton.  

That is a very minor quibble and I have no problem with leaving it as is, so I don't plan to ask for changes.

Reply:
The input is a Boolean so it should start with `is`, `has`, `should`. Also, the previous code version statement `const disableBtn = !!(disableSaveButton && disableSaveButton === true);`  is confusing. I did leave the `disableBtn` var name, although it is more of a method name not a var name.